sql/src/test/java/org/opensearch/sql/sql/antlr/SQLSyntaxParserTest.java

```diff
@@ -448,6 +466,15 @@ private static Stream<String> generateMatchPhraseQueries() {
     return generateQueries("match_phrase", matchPhraseArgs);
   }
 
+  private static Stream<String> generateMatchPhraseQueryQueries() {
```

Comment:
Do we need both `generateMatchPhraseQueryQueries` and `matchPhraseQueryComplexQueries?